docs/DeveloperGuide.md

```diff
@@ -1185,3 +1185,41 @@ Expected: The most recent window size and location is retained.
        behavior}_
 
 1. _{ more test cases ... }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+In crafting LetsGetHired, a lot of thought was put into delivering a user-centric application that 
+simplifies internship application tracking. We prioritized user experience with a revamped GUI and enhanced 
+internship application organization through features like sort and find. We also provided users the 
+flexibility to attach helpful information to entries, through the introduction of notes.
+
+### Designing Organisation Features
+Designing organizational features in LetsGetHired, such as the Find and Sort commands, presented a notable 
+advancement beyond the capabilities of AB3. While AB3 exclusively features a basic Find function that 
+searches for entries based on matching keywords to names in the address book, in LetsGetHired, the Find 
+command empowers users to conduct searches across various fields associated with internship applications.
+Additionally, the Find command in LetsGetHired accommodates searches across 
+multiple fields simultaneously, enhancing user flexibility.
```

Comment:
Perhaps we could highlight that our `find` command also works with partial match - I think this is a powerful (non-trivial) feature that wasn't easy to implement!